### PR TITLE
Add speaker name

### PR DESCRIPTION
**Pull request by martinbonnin**

### CODE REVIEW

File: app/src/main/java/fr/paug/androidmakers/ui/adapter/ScheduleDayAdapter.java [hunk 1]
```diff
@@ -312,42 +314,53 @@ void bind(@NonNull final ScheduleSession scheduleSession, DaySchedule daySchedul
...
+            }
+
+            Session session = AgendaRepository.getInstance().getSession(scheduleSession.getSessionId());
+            if (session != null && session.speakers != null) {
+                for (int i = 0; i < session.speakers.length; i++) {
+                    final Speaker speaker = AgendaRepository.getInstance().getSpeaker(session.speakers[i]);
+
+                    if (speaker != null) {
+                        descriptionBuilder.append('\n');
+                        descriptionBuilder.append(speaker.getFullNameAndCompany());
```
Comment (benju69) [inline app/src/main/java/fr/paug/androidmakers/ui/adapter/ScheduleDayAdapter.java]: I think we don't need the company name

File: gradle/wrapper/gradle-wrapper.properties [hunk 2]
```diff
@@ -2,4 +2,4 @@ distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
 zipStoreBase=GRADLE_USER_HOME
 zipStorePath=wrapper/dists
-distributionUrl=https\://services.gradle.org/distributions/gradle-4.6-all.zip
+distributionUrl=https\://services.gradle.org/distributions/gradle-4.5.1-all.zip
```
Comment (benju69) [inline gradle/wrapper/gradle-wrapper.properties]: Why using a old version of gradle? The 4.7 is just available!

Comment (martinbonnin) [reply gradle/wrapper/gradle-wrapper.properties]: Yes but no: Anyone else (using Gradle 4.6+) getting the follo...

https://www.reddit.com/r/androiddev/comments/8ed6d3/android_studio_312_bugfix_update_now_available/dxueu6j?utm_source=reddit-android

File: app/src/main/java/fr/paug/androidmakers/ui/adapter/ScheduleDayAdapter.java (hunk 1 above)
Comment (martinbonnin) [reply app/src/main/java/fr/paug/androidmakers/ui/adapter/ScheduleDayAdapter.java]: I like information dense screens but fine with me

**Pull request merged**